tests/core/test_read_manifest.py

```diff
@@ -62,6 +51,7 @@ def test_modified_schema(datadir):
 @pytest.mark.nostrict
 def test_last_schema(datadir):
     chip = siliconcompiler.Chip('test')
+    chip.schema._remove('dependency', 'siliconcompiler')
```

Comment:
Same thing
This seems to be a bit the idea of this test but I know it fails with the dependency....